app/views/membership_applications/_membership_application_fields.html.haml

```diff
@@ -5,11 +5,15 @@
       = f.label :membership_number, t('membership_applications.show.membership_number'), class: 'required'
       = f.text_field :membership_number, class: 'wpcf7-form-control'
 
-    = f.label :first_name, t('membership_applications.show.first_name'), class: 'required'
-    = f.text_field :first_name, class: 'wpcf7-form-control'
 
-    = f.label :last_name, t('membership_applications.show.last_name'), class: 'required'
-    = f.text_field :last_name, class: 'wpcf7-form-control'
+    = f.fields_for :user do |ff|
+      = ff.label :first_name, t('membership_applications.show.first_name'), class: 'required'
+      = ff.text_field :first_name, class: 'wpcf7-form-control'
+
+
+      = ff.label :last_name, t('membership_applications.show.last_name'), class: 'required'
+      = ff.text_field :last_name, class: 'wpcf7-form-control'
```

Comment:
The idea is to keep with the current practice: The admin creates a user with only the email address, and the user then supplements firstname and lastname when filling in the application.

I agree that this should be changed in favour of having the user fill in a user profile separately, but those changes haven't been agreed yet by the client.